Description: Remove use-intrinsics feature
 Requires stdsimd, which was removed from Rust in
 https://github.com/rust-lang/rust/commit/ea37e8091fe87ae0a7e204c034e7d55061e56790.
 .
 Can be dropped once we're on half 2.3.0 (see
 https://github.com/starkat99/half-rs/commit/b51e70ad84417206765b4562d5e5becb79dd4539).
Author: Colin Watson <cjwatson@debian.org>
Forwarded: not-needed
Last-Update: 2024-09-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -104,7 +104,6 @@
     "dep:rand_distr",
 ]
 std = ["alloc"]
-use-intrinsics = []
 
 [target."cfg(target_arch = \"spirv\")".dependencies.crunchy]
 version = "0.2.2"
